homography_mapping: [[0,1]]
